### PR TITLE
Removes broken external links.

### PR DESCRIPTION
**Pull request by johnnymatthews**

### CODE REVIEW

File: docs/about-filecoin/how-filecoin-works.md [hunk 1]
```diff
@@ -98,14 +98,12 @@ In Filecoin, addresses are used to identify actors. There are 4 address types:
 
 All actors have an ID such as `99`, a short integer assigned to it by the InitActor when the actor is created. An _ID address_ is an actor's ID prefixed with the network (`f` for mainnet) and address type `0` for ID.
 
-For example, the [Burn Account](https://filfox.info/en/address/f099) actor on the mainnet has the ID `99` and the ID Address `f099`.
+For example, the _burn account actor on the mainnet has the ID `99` and the ID Address `f099`.
```
Comment (johnnymatthews) [inline docs/about-filecoin/how-filecoin-works.md:101]:
```suggestion
For example, the _burn account_ actor on the mainnet has the ID `99` and the ID Address `f099`.
```

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/about-filecoin/how-filecoin-works.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://www.parity.io/why-libp2p/
- The following link returned HTTP status code 303 (See Other):
  https://youtu.be/SXlTBvcqzz8
- This phrase is duplicated. You should probably use “Filecoin nodes” only once.
  ```
  ... many thousands of peers participate. # Filecoin nodes Filecoin Nodes or Fil...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...unning a Filecoin storage provider is a highly-technical task with strong ha...
                                             ^^^^^^^^^^^^^^^^
  ```
- ‘to this day’ might be wordy. Consider a shorter alternative.
  ```
  ...entation of a Filecoin storage provider to this day. # Deals There are two m...
                                             ^^^^^^^^^^^
  ```
- Consider adding a comma after ‘Historically’ for more clarity.
  ```
  ...d in a new block by a storage provider. Historically in other blockchains, m...
                                             ^^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...reum's EIP1559  (opens new window). The amount of fees burned in the Filecoi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...ecoin network comes given by a dynamic *BaseFee which gets automatically adj...
                                             ^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...ached to each message and determine the amount of rewards that storage provi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...mated (see prices  (opens new window)). GasUsage measured in units of Gas. B...
                                             ^^^^^^^^
  ```
- Unpaired symbol: ‘(’ seems to be missing
  ```
  ...rrent implementation  (opens new window)). The total cost of a message for a...
                                             ^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...s, you can end up referring to the same id for different rows. The Expected ...
                                             ^^
  ```
- Possible missing comma found.
  ```
  ...coin uses G1 for public keys and G2 for signatures as G1 allows for a smalle...
                                             ^^^^^^^^^^
  ```
- ‘is in contrast to’ might be wordy. Consider a shorter alternative.
  ```
  ...ivate keys in little-endian order. This is in contrast to ETH2 keys, which a...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Possible typo detected.
  ```
  ...is built on top of mature projects like libp2p (networking, addressing, mess...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...otoSchool tutorials  (opens new window) libp2p: Why libp2p?  (opens new wind...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...torials  (opens new window) libp2p: Why libp2p?  (opens new window) libp2p: ...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...libp2p: Why libp2p?  (opens new window) libp2p: A Modular, p2p Networking St...
                                             ^^^^^^
  ```
### `docs/community/contribute/contribution-tutorial.md`:
- The following link returned HTTP status code 302 (Found):
  https://github.com/notifications
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  # Contribution tutorial While the grammar, formatting, and style and ...
                          ^^^^^
  ```
- “you” seems less likely than “your” (belonging to you).
  ```
  ... delve into how you can actually submit you content changes. This guide will...
                                             ^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...w. Merging your fix. This may look like a lot of steps for a small issue fix...
                                             ^^^^^^^^
  ```
- Consider using the plural verb form for the plural noun “reasons”.
  ```
  ...e Filecoin project is hosted in GitHub. There's a bunch of reasons for this,...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s, find issues, and raise issues in the docs repository (called a repo for s...
                                             ^^^^
  ```
- The phrase ‘lots of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...tell from the available tags, there are lots of different types of issues. S...
                                             ^^^^^^^
  ```
- If the ‘because’ clause is essential to the meaning, do not use a comma before the clause.
  ```
  ...y changes to this copy whenever you want, because you own it. The idea is th...
                                             ^^^^^^^^^
  ```
### `docs/community/contribute/grammar-formatting-and-style.md`:
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://www.grammarly.com/
- The following link returned HTTP status code 302 (Found):
  https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md
- Possible missing comma found.
  ```
  ... when referring to the Lotus executable commands such as lotus daemon. Lotus...
                                             ^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...ion: This generally isn't necessary for commonly-encountered acronyms like I...
                                             ^^^^^^^^^^^^^^^^^^^^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ...he GitHub Flavoured Markdown syntax for markdown. This way, all articles dis...
                                             ^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ders the option to view articles on the docs website  (opens new window) or ...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...ter title and description: In the above example title: serves as a <h1> or #...
                                             ^^^^^^^
  ```
- Unpaired symbol: ‘"’ seems to be missing
  ```
  ...italics and surrounded by double quotes ": # Code blocks Tag code blocks wit...
                                             ^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ... with-output tag that can be applied to markdown code blocks to support comm...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...e applied correctly. # Inline code tags Surround directories, file names, an...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...rom Father Christmas. # Unordered lists Use the dash character - for un-numb...
                                             ^^^
  ```
- This phrase is redundant. Consider using “outside”.
  ```
  ...s in code tags: The plus symbol + stays outside of the code tags. # Custom c...
                                             ^^^^^^^^^^
  ```
- This phrase is duplicated. You should probably use “Custom containers” only once.
  ```
  ...bol + stays outside of the code tags. # Custom containers Custom containers ...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The official name of this software platform is spelled with a capital “H”.
  ```
  ... Output This output is not supported by Github Flavoured Markdown (GFM). How...
                                             ^^^^^^
  ```
- After the expression ‘for example’ a comma is usually used.
  ```
  ...der within images for each article. For example the article proof-of-spaceti...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ticle, so there is no folder within the images directory for that article. #...
                                             ^^^^^^
  ```
### `docs/community/contribute/ways-to-contribute.md`:
- You have already used this phrasing in nearby sentences. Consider replacing it to add variety to your writing.
  ```
  ...w you can get started. Even if what you want to do is not listed here, we pr...
                                             ^^^^^^^
  ```
- Use “an” instead of ‘a’ if the following word starts with a vowel sound, e.g. ‘an article’, ‘an hour’.
  ```
  ...s for it! If you're unsure, please open a issue. # Code Filecoin and its sis...
                                             ^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...ritten in multiple languages. We always need help writing and maintaining co...
                                             ^^^^
  ```
- Possible typo detected.
  ```
  ...ated projects: IPFS  (opens new window) libp2p  (opens new window) IPLD  (op...
                                             ^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...lots of good documentation! However, we need a lot more help to write the aw...
                                             ^^^^
  ```
- Consider using a more formal and expressive alternative to the somewhat overused ‘awesome’.
  ```
  ...r, we need a lot more help to write the awesome docs the project needs. If w...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... consistent! Style and formatting guide Writing guide If you have never cont...
                                             ^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ... an open-source project before, or just need a refresher, take a look at the...
                                             ^^^^
  ```
- Using many exclamation marks might seem excessive (in this case: 5 exclamation marks for a text that’s 3004 characters long)
  ```
  ...oin. Build anything you think is missing! If you're unsure about something, ...
                                             ^
  ```
### `docs/community/contribute/writing-guide.md`:
- Possible missing comma found.
  ```
  ...g contains little to no emotion. Intent Instruct Tell the reader how to do s...
                                             ^^^^^^^^
  ```
- Make sure that the ambiguous verb form ‘span’ is correct. (It can either be the base form ‘span’, or the past tense of a different verb.)
  ```
  ...onceptual articles. Most tutorials will span several pages, and contain mult...
                                             ^^^^
  ```
- Possible missing preposition found.
  ```
  ...in them. Take the hypothetical tutorial Get up and running with Filecoin, fo...
                                             ^^^
  ```
### `docs/community/social-media/social-media.md`:
- The following link returned HTTP status code 302 (Found):
  https://filecoin.io/newsletter
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://weixin.qq.com/r/1xz54Y-EctINrcuC90nF

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/about-filecoin/how-filecoin-works.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://www.parity.io/why-libp2p/
- The following link returned HTTP status code 303 (See Other):
  https://youtu.be/SXlTBvcqzz8
- This phrase is duplicated. You should probably use “Filecoin nodes” only once.
  ```
  ... many thousands of peers participate. # Filecoin nodes Filecoin Nodes or Fil...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...unning a Filecoin storage provider is a highly-technical task with strong ha...
                                             ^^^^^^^^^^^^^^^^
  ```
- ‘to this day’ might be wordy. Consider a shorter alternative.
  ```
  ...entation of a Filecoin storage provider to this day. # Deals There are two m...
                                             ^^^^^^^^^^^
  ```
- Consider adding a comma after ‘Historically’ for more clarity.
  ```
  ...d in a new block by a storage provider. Historically in other blockchains, m...
                                             ^^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...reum's EIP1559  (opens new window). The amount of fees burned in the Filecoi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...ecoin network comes given by a dynamic *BaseFee which gets automatically adj...
                                             ^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...ached to each message and determine the amount of rewards that storage provi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...mated (see prices  (opens new window)). GasUsage measured in units of Gas. B...
                                             ^^^^^^^^
  ```
- Unpaired symbol: ‘(’ seems to be missing
  ```
  ...rrent implementation  (opens new window)). The total cost of a message for a...
                                             ^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...s, you can end up referring to the same id for different rows. The Expected ...
                                             ^^
  ```
- Possible missing comma found.
  ```
  ...coin uses G1 for public keys and G2 for signatures as G1 allows for a smalle...
                                             ^^^^^^^^^^
  ```
- ‘is in contrast to’ might be wordy. Consider a shorter alternative.
  ```
  ...ivate keys in little-endian order. This is in contrast to ETH2 keys, which a...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Possible typo detected.
  ```
  ...is built on top of mature projects like libp2p (networking, addressing, mess...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...otoSchool tutorials  (opens new window) libp2p: Why libp2p?  (opens new wind...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...torials  (opens new window) libp2p: Why libp2p?  (opens new window) libp2p: ...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...libp2p: Why libp2p?  (opens new window) libp2p: A Modular, p2p Networking St...
                                             ^^^^^^
  ```
### `docs/community/contribute/contribution-tutorial.md`:
- The following link returned HTTP status code 302 (Found):
  https://github.com/notifications
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  # Contribution tutorial While the grammar, formatting, and style and ...
                          ^^^^^
  ```
- “you” seems less likely than “your” (belonging to you).
  ```
  ... delve into how you can actually submit you content changes. This guide will...
                                             ^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...w. Merging your fix. This may look like a lot of steps for a small issue fix...
                                             ^^^^^^^^
  ```
- Consider using the plural verb form for the plural noun “reasons”.
  ```
  ...e Filecoin project is hosted in GitHub. There's a bunch of reasons for this,...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s, find issues, and raise issues in the docs repository (called a repo for s...
                                             ^^^^
  ```
- The phrase ‘lots of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...tell from the available tags, there are lots of different types of issues. S...
                                             ^^^^^^^
  ```
- If the ‘because’ clause is essential to the meaning, do not use a comma before the clause.
  ```
  ...y changes to this copy whenever you want, because you own it. The idea is th...
                                             ^^^^^^^^^
  ```
### `docs/community/contribute/grammar-formatting-and-style.md`:
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://www.grammarly.com/
- The following link returned HTTP status code 302 (Found):
  https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md
- Possible missing comma found.
  ```
  ... when referring to the Lotus executable commands such as lotus daemon. Lotus...
                                             ^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...ion: This generally isn't necessary for commonly-encountered acronyms like I...
                                             ^^^^^^^^^^^^^^^^^^^^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ...he GitHub Flavoured Markdown syntax for markdown. This way, all articles dis...
                                             ^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ders the option to view articles on the docs website  (opens new window) or ...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...ter title and description: In the above example title: serves as a <h1> or #...
                                             ^^^^^^^
  ```
- Unpaired symbol: ‘"’ seems to be missing
  ```
  ...italics and surrounded by double quotes ": # Code blocks Tag code blocks wit...
                                             ^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ... with-output tag that can be applied to markdown code blocks to support comm...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...e applied correctly. # Inline code tags Surround directories, file names, an...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...rom Father Christmas. # Unordered lists Use the dash character - for un-numb...
                                             ^^^
  ```
- This phrase is redundant. Consider using “outside”.
  ```
  ...s in code tags: The plus symbol + stays outside of the code tags. # Custom c...
                                             ^^^^^^^^^^
  ```
- This phrase is duplicated. You should probably use “Custom containers” only once.
  ```
  ...bol + stays outside of the code tags. # Custom containers Custom containers ...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The official name of this software platform is spelled with a capital “H”.
  ```
  ... Output This output is not supported by Github Flavoured Markdown (GFM). How...
                                             ^^^^^^
  ```
- After the expression ‘for example’ a comma is usually used.
  ```
  ...der within images for each article. For example the article proof-of-spaceti...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ticle, so there is no folder within the images directory for that article. #...
                                             ^^^^^^
  ```
### `docs/community/contribute/ways-to-contribute.md`:
- You have already used this phrasing in nearby sentences. Consider replacing it to add variety to your writing.
  ```
  ...w you can get started. Even if what you want to do is not listed here, we pr...
                                             ^^^^^^^
  ```
- Use “an” instead of ‘a’ if the following word starts with a vowel sound, e.g. ‘an article’, ‘an hour’.
  ```
  ...s for it! If you're unsure, please open a issue. # Code Filecoin and its sis...
                                             ^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...ritten in multiple languages. We always need help writing and maintaining co...
                                             ^^^^
  ```
- Possible typo detected.
  ```
  ...ated projects: IPFS  (opens new window) libp2p  (opens new window) IPLD  (op...
                                             ^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...lots of good documentation! However, we need a lot more help to write the aw...
                                             ^^^^
  ```
- Consider using a more formal and expressive alternative to the somewhat overused ‘awesome’.
  ```
  ...r, we need a lot more help to write the awesome docs the project needs. If w...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... consistent! Style and formatting guide Writing guide If you have never cont...
                                             ^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ... an open-source project before, or just need a refresher, take a look at the...
                                             ^^^^
  ```
- Using many exclamation marks might seem excessive (in this case: 5 exclamation marks for a text that’s 3004 characters long)
  ```
  ...oin. Build anything you think is missing! If you're unsure about something, ...
                                             ^
  ```
### `docs/community/contribute/writing-guide.md`:
- Possible missing comma found.
  ```
  ...g contains little to no emotion. Intent Instruct Tell the reader how to do s...
                                             ^^^^^^^^
  ```
- Make sure that the ambiguous verb form ‘span’ is correct. (It can either be the base form ‘span’, or the past tense of a different verb.)
  ```
  ...onceptual articles. Most tutorials will span several pages, and contain mult...
                                             ^^^^
  ```
- Possible missing preposition found.
  ```
  ...in them. Take the hypothetical tutorial Get up and running with Filecoin, fo...
                                             ^^^
  ```
### `docs/community/social-media/social-media.md`:
- The following link returned HTTP status code 302 (Found):
  https://filecoin.io/newsletter
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://weixin.qq.com/r/1xz54Y-EctINrcuC90nF

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!
### `docs/about-filecoin/how-filecoin-works.md`:
- The following link returned HTTP status code 301 (Moved Permanently):
  https://www.parity.io/blog/why-libp2p
- This phrase is duplicated. You should probably use “Filecoin nodes” only once.
  ```
  ... many thousands of peers participate. # Filecoin nodes Filecoin Nodes or Fil...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...unning a Filecoin storage provider is a highly-technical task with strong ha...
                                             ^^^^^^^^^^^^^^^^
  ```
- ‘to this day’ might be wordy. Consider a shorter alternative.
  ```
  ...entation of a Filecoin storage provider to this day. # Deals There are two m...
                                             ^^^^^^^^^^^
  ```
- Consider adding a comma after ‘Historically’ for more clarity.
  ```
  ...d in a new block by a storage provider. Historically in other blockchains, m...
                                             ^^^^^^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...reum's EIP1559  (opens new window). The amount of fees burned in the Filecoi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...ecoin network comes given by a dynamic *BaseFee which gets automatically adj...
                                             ^^^^^^^
  ```
- ‘Amount of’ should usually only be used with uncountable or mass nouns. Consider using “number” if this is not the case.
  ```
  ...ached to each message and determine the amount of rewards that storage provi...
                                             ^^^^^^
  ```
- Possible missing comma found.
  ```
  ...mated (see prices  (opens new window)). GasUsage measured in units of Gas. B...
                                             ^^^^^^^^
  ```
- Unpaired symbol: ‘(’ seems to be missing
  ```
  ...rrent implementation  (opens new window)). The total cost of a message for a...
                                             ^
  ```
- This abbreviation for “identification” is spelled all-uppercase.
  ```
  ...s, you can end up referring to the same id for different rows. The Expected ...
                                             ^^
  ```
- Possible missing comma found.
  ```
  ...coin uses G1 for public keys and G2 for signatures as G1 allows for a smalle...
                                             ^^^^^^^^^^
  ```
- ‘is in contrast to’ might be wordy. Consider a shorter alternative.
  ```
  ...ivate keys in little-endian order. This is in contrast to ETH2 keys, which a...
                                             ^^^^^^^^^^^^^^^^^
  ```
- Possible typo detected.
  ```
  ...is built on top of mature projects like libp2p (networking, addressing, mess...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...otoSchool tutorials  (opens new window) libp2p: Why libp2p?  (opens new wind...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...torials  (opens new window) libp2p: Why libp2p?  (opens new window) libp2p: ...
                                             ^^^^^^
  ```
- Possible typo detected.
  ```
  ...libp2p: Why libp2p?  (opens new window) libp2p: A Modular, p2p Networking St...
                                             ^^^^^^
  ```
### `docs/community/contribute/contribution-tutorial.md`:
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  # Contribution tutorial While the grammar, formatting, and style and ...
                          ^^^^^
  ```
- “you” seems less likely than “your” (belonging to you).
  ```
  ... delve into how you can actually submit you content changes. This guide will...
                                             ^^^
  ```
- The phrase ‘a lot of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...w. Merging your fix. This may look like a lot of steps for a small issue fix...
                                             ^^^^^^^^
  ```
- Consider using the plural verb form for the plural noun “reasons”.
  ```
  ...e Filecoin project is hosted in GitHub. There's a bunch of reasons for this,...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...s, find issues, and raise issues in the docs repository (called a repo for s...
                                             ^^^^
  ```
- The phrase ‘lots of’ might be wordy. Consider using “many”, “numerous” or “countless”.
  ```
  ...tell from the available tags, there are lots of different types of issues. S...
                                             ^^^^^^^
  ```
- If the ‘because’ clause is essential to the meaning, do not use a comma before the clause.
  ```
  ...y changes to this copy whenever you want, because you own it. The idea is th...
                                             ^^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...p returning to GitHub and checking your notifications page to make sure you ...
                                             ^^^^^^^^^^^^^
  ```
### `docs/community/contribute/grammar-formatting-and-style.md`:
- The following link returned HTTP status code 405 (Method Not Allowed):
  https://www.grammarly.com/
- The following link returned HTTP status code 302 (Found):
  https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md
- Possible missing comma found.
  ```
  ... when referring to the Lotus executable commands such as lotus daemon. Lotus...
                                             ^^^^^^^^
  ```
- Although a hyphen is possible, it is not necessary in a compound modifier in which the first word is an adverb that ends in ‘ly’.
  ```
  ...ion: This generally isn't necessary for commonly-encountered acronyms like I...
                                             ^^^^^^^^^^^^^^^^^^^^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ...he GitHub Flavoured Markdown syntax for markdown. This way, all articles dis...
                                             ^^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ders the option to view articles on the docs website  (opens new window) or ...
                                             ^^^^
  ```
- Possible missing comma found.
  ```
  ...ter title and description: In the above example title: serves as a <h1> or #...
                                             ^^^^^^^
  ```
- Unpaired symbol: ‘"’ seems to be missing
  ```
  ...italics and surrounded by double quotes ": # Code blocks Tag code blocks wit...
                                             ^
  ```
- Did you mean the formatting language “Markdown” (= proper noun)?
  ```
  ... with-output tag that can be applied to markdown code blocks to support comm...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...e applied correctly. # Inline code tags Surround directories, file names, an...
                                             ^^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ...rom Father Christmas. # Unordered lists Use the dash character - for un-numb...
                                             ^^^
  ```
- This phrase is redundant. Consider using “outside”.
  ```
  ...s in code tags: The plus symbol + stays outside of the code tags. # Custom c...
                                             ^^^^^^^^^^
  ```
- This phrase is duplicated. You should probably use “Custom containers” only once.
  ```
  ...bol + stays outside of the code tags. # Custom containers Custom containers ...
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ```
- The official name of this software platform is spelled with a capital “H”.
  ```
  ... Output This output is not supported by Github Flavoured Markdown (GFM). How...
                                             ^^^^^^
  ```
- After the expression ‘for example’ a comma is usually used.
  ```
  ...der within images for each article. For example the article proof-of-spaceti...
                                             ^^^^^^^
  ```
- An apostrophe may be missing.
  ```
  ...ticle, so there is no folder within the images directory for that article. #...
                                             ^^^^^^
  ```
### `docs/community/contribute/ways-to-contribute.md`:
- You have already used this phrasing in nearby sentences. Consider replacing it to add variety to your writing.
  ```
  ...w you can get started. Even if what you want to do is not listed here, we pr...
                                             ^^^^^^^
  ```
- Use “an” instead of ‘a’ if the following word starts with a vowel sound, e.g. ‘an article’, ‘an hour’.
  ```
  ...s for it! If you're unsure, please open a issue. # Code Filecoin and its sis...
                                             ^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...ritten in multiple languages. We always need help writing and maintaining co...
                                             ^^^^
  ```
- Possible typo detected.
  ```
  ...ated projects: IPFS  (opens new window) libp2p  (opens new window) IPLD  (op...
                                             ^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ...lots of good documentation! However, we need a lot more help to write the aw...
                                             ^^^^
  ```
- Consider using a more formal and expressive alternative to the somewhat overused ‘awesome’.
  ```
  ...r, we need a lot more help to write the awesome docs the project needs. If w...
                                             ^^^^^^^
  ```
- Only proper nouns start with an uppercase character (there are exceptions for headlines).
  ```
  ... consistent! Style and formatting guide Writing guide If you have never cont...
                                             ^^^^^^^
  ```
- This word has been used in one of the immediately preceding sentences. Using a synonym could make your text more interesting to read, unless the repetition is intentional.
  ```
  ... an open-source project before, or just need a refresher, take a look at the...
                                             ^^^^
  ```
- Using many exclamation marks might seem excessive (in this case: 5 exclamation marks for a text that’s 3004 characters long)
  ```
  ...oin. Build anything you think is missing! If you're unsure about something, ...
                                             ^
  ```
### `docs/community/contribute/writing-guide.md`:
- Possible missing comma found.
  ```
  ...g contains little to no emotion. Intent Instruct Tell the reader how to do s...
                                             ^^^^^^^^
  ```
- Make sure that the ambiguous verb form ‘span’ is correct. (It can either be the base form ‘span’, or the past tense of a different verb.)
  ```
  ...onceptual articles. Most tutorials will span several pages, and contain mult...
                                             ^^^^
  ```
- Possible missing preposition found.
  ```
  ...in them. Take the hypothetical tutorial Get up and running with Filecoin, fo...
                                             ^^^
  ```
- Check of `docs/community/social-media/social-media.md` came back clean!

File: docs/networks/README.md [hunk 2]
```diff
@@ -120,7 +120,6 @@ Calibration network is the most realistic simulation of the Filecoin mainnet:
 - [Stats Dashboard](https://stats.calibration.fildev.network/)
 - [Slack Channel for Updates: #fil-net-calibration-announce](https://filecoinproject.slack.com/archives/C01C5PT7ETC)
 - [Slack Channel for Questions: #fil-net-calibration-discuss](https://filecoinproject.slack.com/archives/C01D42NNLMS)
-- [Block explorer - Filscout for Calibration](https://calibration.filscout.com/en)
```
Comment (johnnymatthews) [inline docs/networks/README.md:123]: Security warning. Removed link.

File: docs/reference/README.md [hunk 3]
```diff
@@ -97,7 +97,6 @@ Tools to check status and details of the network and chain.
 
 - [Filstats.io](https://filstats.io) - node telemetry monitoring - add your node!
 - [Sentinel Project](https://github.com/filecoin-project/sentinel) - Filecoin Network Monitoring and Analysis System behind Grafana.
-- [Statediff](https://node.glif.io/space07/statediff/rpc/) - exploration and comparing changes to Filecoin state.
```
Comment (johnnymatthews) [inline docs/reference/README.md:100]: 404ed.

**Pull request merged**